### PR TITLE
Replaced ekirjasto login error message strings

### PR DESCRIPTION
**Pull request by mikko-work**
**What's this do?**
Added new error messages to ekirjasto login.

**Why are we doing this? (w/ JIRA link if applicable)**
[Quick blurb about why the code is needed and Jira link goes here / Do these changes meet the business requirements of the story?]

**How should this be tested? / Do these changes have associated tests?**
[Description of any tests that need to be performed once merged goes here]

**Dependencies for merging? Releasing to production?**
[Description of any watchouts, dependencies, or issues we should be aware of goes here]

**Have you updated the changelog?**

**Has the application documentation been updated for these changes?**

**Did someone actually run this code to verify it works?**

### CODE REVIEW

File: simplified-ui-accounts/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -80,4 +80,8 @@
   <string name="account_sync_bookmarks_statement">Save your reading position and bookmarks to all your other devices</string>
   <string name="e_kirjasto_logo">E-Kirjasto logo</string>
   <string name="natlibfi_logo">National Library of Finland logo</string>
+  <string name="errorLoginFailed">Could not log in</string>
```
Comment (sylvain-degeilh-lingsoft) [inline simplified-ui-accounts/src/main/res/values/strings.xml:83]: Could not sign in

File: simplified-ui-accounts/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -80,4 +80,8 @@
...
+  <string name="errorPasskeyLoginFailed">Could not sign in with Passkey</string>
+  <string name="errorPasskeyRegisterFailed">Error Registering a Passkey</string>
+  <string name="errorSuomiFiLoginFailed">Could not sign in with suomi.fi</string>
```
Comment (sylvain-degeilh-lingsoft) [inline simplified-ui-accounts/src/main/res/values/strings.xml:86]: Suomi.fi

**Review by sylvain-degeilh-lingsoft — CHANGES_REQUESTED** [review]

File: simplified-ui-accounts/src/main/res/values/strings.xml (hunk 1 above)
Comment (kankaristo) [reply simplified-ui-accounts/src/main/res/values/strings.xml]: Mikko updated this in the latest commit.

File: simplified-ui-accounts/src/main/res/values/strings.xml (hunk 2 above)
Comment (kankaristo) [reply simplified-ui-accounts/src/main/res/values/strings.xml]: Mikko updated this in the latest commit.

**Review by kankaristo — APPROVED** [review]
Looks good to me, the changes requested by @sylvain-degeilh-lingsoft were made.

**Pull request merged**